elfi/visualization.py

```diff
@@ -1,3 +1,7 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from collections import OrderedDict
+
 from elfi.model.elfi_model import ElfiModel, NodeReference
 
 
```

Comment:
Could this file be combined with `plotting.py` so that we don't have two different files for plotting.